conans/client/conf/config_installer.py

```diff
@@ -247,3 +251,27 @@ def configuration_install(app, uri, verify_ssl, config_type=None,
         else:
             configs = [(c if c != config else config) for c in configs]
         _save_configs(configs_file, configs)
+
+
+def is_config_install_scheduled(api):
+    """ Validate if the next config install is scheduled to occur now
+
+        When config_install_interval is not configured, config install should not run
+        When config_install_interval is configured, config install will respect the delta from:
+            last conan install execution (sched file) + config_install_interval value < now
+
+    :param api: Conan API instance
+    :return: True, if it should occur now. Otherwise, False.
+    """
+    cache = ClientCache(api.cache_folder, api.out)
+    interval = cache.config.config_install_interval
+    config_install_file = cache.config_install_file
+    if interval is not None:
+        if os.path.exists(config_install_file):
+            timestamp = os.path.getmtime(config_install_file)
+            sched = datetime.fromtimestamp(timestamp, tz=gettz())
+            sched += interval
+            now = datetime.now(gettz())
+            return now > sched
+        else:
+            raise ConanException("config_install_interval defined, but no config_install file")
```

Comment:
@uilianries: this is a weird case, we don't want to have an interval defined without a file that tells what needs to be ``conan config install``